maro/cli/maro.py

```diff
@@ -827,6 +835,31 @@ def load_parser_meta(prev_parser: ArgumentParser, global_parser: ArgumentParser)
     deploy_cmd_parser.set_defaults(func=meta_deploy)
 
 
+def load_parser_inspector(prev_parser: ArgumentParser, global_parser: ArgumentParser):
+    inspector_cmd_sub_parsers = prev_parser.add_subparsers()
+
+    from maro.cli.inspector.env_data_process import start_vis
```

Comment:
why not import at the beginning of this file?

Reply:
I followed the custom of other functions. Do all functions in this file need to be changed together? Maybe we can make sure with them when we have a group meeting tomorrow?